pandas/io/sql.py

```diff
@@ -2373,6 +2399,14 @@ def to_sql(
                     cur.execute(f"DROP TABLE {table_name}")
```

Comment:
I was wondering if we should use `mode=replace` instead of running a manual `DROP TABLE` statement. I can have other PR for this patch if it makes sense...